test/GlobStream.js

```diff
@@ -27,12 +17,38 @@ describe('GlobStream', function() {
     finished(
       gs,
       function(err) {
-        expect(arguments[0].message).toMatch(/^File not found with singular glob/g);
+        expect(err.message).toMatch(/^File not found with singular glob/g);
         done();
       }
     );
   });
 
+  it('throws an error if you try to write to it', function(done) {
+    var gs = stream(
+      './fixtures/whatsgoingon/lkasdkl.txt',
+      [],
+      { cwd: __dirname, }
+    );
+
+    try {
+      gs.write({});
+    } catch (err) {
+      expect(err.message).toExist();
+      done();
+    }
+  });
+
+  it('does not throw an error if you try to push to it', function(done) {
+    var gs = stream(
+      './fixtures/whatsgoingon/lkasdkl.txt',
+      [],
+      { cwd: __dirname, }
+    );
+
+    gs.push({});
```

Comment:
Ahh gotya, yeah I was just looking it over last night before calling it haha. Sounds good!